package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Stupidly, this file is 100% necessary because the plugin looks for its existence and errors out if it isn't there.. So yay.